src/pages/ScholarshipForm.jsx

```diff
@@ -19,8 +20,19 @@ const useStyles = makeStyles((theme) => ({
 
 function ScholarshipForm() {
   const classes = useStyles();
-  const [submissionAlert, setSubmissionAlert] = useState(false);
-  const [error, setError] = useState(false);
+  const [showAlert, setShowAlert] = useState(false);
```

Comment:
You can derive `showAlert` from just whether `submissionAlert` is set and set `submissionAlert` to null when there's nothing to show.